### ISSUE TITLE
Detected package downgrade: System.Runtime.CompilerServices.Unsafe

### ISSUE THREAD

**Issue by krombipils** [eXpandFramework/eXpand#559]
In 19.1.602.10 there is an issue on nuget restore, if you reference eXpandIOWin:

NU1605: Detected package downgrade: System.Runtime.CompilerServices.Unsafe from 4.6.0 to 4.5.2. Reference the package directly from the project to select a different version.

WCFWebAppServer -> WCFCore -> eXpandIOWin 19.1.602.10 -> eXpandTreelistViewWin 19.1.602.10 -> System.Runtime.CompilerServices.Unsafe (>= 4.6.0)

WCFWebAppServer -> WCFCore -> eXpandIOWin 19.1.602.10 -> System.Runtime.CompilerServices.Unsafe (>= 4.5.2)

When looking at the nuget packages in NPE, there is a direct reference to System.Runtime.CompilerServices.Unsafe (>= 4.5.2) in eXpandIOWin and a transitive reference to System.Runtime.CompilerServices.Unsafe (>= 4.6.0) via eXpandTreelistViewWin.

**Comment by krombipils** [comment]
Of course the issue can be fixed by following the suggestion (add reference to System.Runtime.CompilerServices.Unsafe 4.6.0), but I think it does make sense to consolidate package references in eXpand.

**Comment by apobekiaris** [comment]
yes i am workling on it, does not even make sense to have a direct dependency to this package

**Comment by apobekiaris** [comment]
so time for an elab: the System.Runtime.CompilerServices.Unsafe is used from the System :). Meaning many system packages consume it with different versions. Xpand builds are configured to fail on Warning so I added this dependency to all projects that had a warning. I will update all consumers to 4.6.0 for now

**Comment by apobekiaris** [comment]
it looks like projects configurations like 
```
<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
```
fail to remove the warnings

**Comment by krombipils** [comment]
> I will update all consumers to 4.6.0 

I'm using variables in directory.build.props to consolidate references used in multiple projects...

**Comment by apobekiaris** [comment]
now i already powershell it and to my understand it the props file will add a dependency to all projects??

**Comment by apobekiaris** [comment]
can u post how can be done with the props?

**Comment by krombipils** [comment]
From my directory.build.props:

```
<Project>
  <PropertyGroup>
   <Deterministic>true</Deterministic>
   <DevExpressVersion>19.1.6</DevExpressVersion>
   <ExpandVersion>19.1.602.10</ExpandVersion>
...
```

From one of my csproj files:
```
...
<ItemGroup>
  <PackageReference Include="eXpandSecurity" Version="$(ExpandVersion)" />
  <PackageReference Include="DevExpress.ExpressApp.ConditionalAppearance" Version="$(DevExpressVersion)" />
```

**Comment by apobekiaris** [comment]
yes is a good option for this case

thnks

**Comment by expand** [comment]
eXpand.lab release [19.1.602.11](https://github.com/eXpandFramework/eXpand.lab/releases/19.1.602.11) includes commit that relate to this task:

* [Bump System.Runtime.CompilerServices.Unsafe #559](https://github.com/eXpandFramework/eXpand.lab/commit/92bdd89ff5f23513b2942b3d02d2cb5d166ae767)

Please test if it addresses the problem. If you use nuget add our LAB [NugetServer](https://xpandnugetserver.azurewebsites.net/nuget) as a nuget package source in VS.

Thanks a lot for your contribution.

**Comment by krombipils** [comment]
Problem is fixed in 19.1.602.11. Thank you!
